packages/transaction-controller/src/types.ts

```diff
@@ -645,6 +645,41 @@ export interface TransactionParams {
   type?: string;
 }
 
+/**
+ * Properties that can be updated in a transaction.
+ */
+export type EditableTransactionParams = {
```

Comment:
Do we need this type? I fear we have so many types already that it's adding complexity if we add additional ones that are only used on single methods, especially the update methods as I don't think any others have specific types.